test/powershell/Modules/Microsoft.PowerShell.Utility/Measure-Object.Tests.ps1

```diff
@@ -17,6 +18,25 @@ Describe "Measure-Object" -Tags "CI" {
         $($testObject | Measure-Object).Count | Should Be $testObject.Length
     }
 
+    It "Should calculate Standard Deviation" {
+        $actual = ($testObject | Measure-Object -Average -StdDeviation)
+        # We check this way since .StdDeviation returns a double value
+        [Math]::abs($actual.StdDeviation - 1.52752523165195) | Should -BeLessThan .00000000000001
+    }
+
+
+    It "Should calculate Standard Deviation" {
+        $actual = ($testObject2 | Measure-Object -StdDeviation)
+        # We check this way since .StdDeviation returns a double value
+        [Math]::abs($actual.StdDeviation - 29.011491975882) | Should -BeLessThan .0000000000001
+    }
+
+    It "Should calculate Standard Deviation without -Average" {
+        $actual = ($testObject | Measure-Object -StdDeviation)
```

Comment:
The test is duplicate previous test.
I believe we could chech that `-StdDeviation` works if other parameters present.